src/components/CommentFooter/CommentFooter.less

```diff
@@ -4,6 +4,12 @@
   padding: 8px 0;
   color: @blue-botticelli;
   line-height: 14px;
+  
+  span:nth-child(7) > span{
```

Comment:
idk if this is the right approach.. select the 7th child child then the child span of that.... @Sekhmet what are your thoughts?

Reply:
Yeah, we should have a class for this.